### PR TITLE
added assert function

### PR DESCRIPTION
**Pull request by bekzod**
#489

### CODE REVIEW

File: addon/route-handlers/shorthands/delete.js [hunk 1]
```diff
@@ -13,12 +13,14 @@ export default class DeleteShorthandRouteHandler extends BaseShorthandRouteHandl
   handleStringShorthand(modelName, schema, request) {
     let id = this._getIdForRequest(request);
     let type = camelize(modelName);
+    let schem = schema[type];
```
Comment (samselikoff) [inline addon/route-handlers/shorthands/delete.js]: could we rename `schem` to `modelClass` here (and through the rest of the PR)? I think it's a bit more descriptive

File: addon/route-handlers/shorthands/post.js [hunk 2]
```diff
@@ -13,10 +13,12 @@ export default class PostShorthandRouteHandler extends BaseShorthandRouteHandler
...
 
-    if (!schema[type]) {
-      throw new MirageError(`The route handler for ${request.url} is trying to access the ${type} model, but that model doesn't exist. Create it using 'ember g mirage-model ${modelName}'.`);
-    }
+    assert(
+      schem,
+      `The route handler for ${request.url} is trying to access the ${type} model, but that model doesn't exist. Create it using 'ember g mirage-model ${modelName}'.`
+    );
 
     return schema[type].create(attrs);
```
Comment (samselikoff) [inline addon/route-handlers/shorthands/post.js]: use `modelClass` here

**Comment by samselikoff** [comment]
This looks great! Just the small changes noted above and I'll merge this in.

**Comment by bekzod** [comment]
thanks, fixed those things

File: addon/route-handlers/shorthands/delete.js [hunk 3]
```diff
@@ -13,12 +13,14 @@ export default class DeleteShorthandRouteHandler extends BaseShorthandRouteHandl
   handleStringShorthand(modelName, schema, request) {
     let id = this._getIdForRequest(request);
     let type = camelize(modelName);
+    let modelClass = schema[type];
 
-    if (!schema[type]) {
```
Comment (samselikoff) [inline addon/route-handlers/shorthands/delete.js]: you could use `modelClass` in the asserts too! :D

Comment (bekzod) [inline addon/route-handlers/shorthands/delete.js]: you mean in assert text, instead of `modelName` to use `modelClass ` ?

Comment (samselikoff) [inline addon/route-handlers/shorthands/delete.js]: oh woops worry, was looking at the old code. this looks great!

**Comment by samselikoff** [comment]
:+1: thank you!

**Pull request merged**